companies/bigpoint.json

```diff
@@ -0,0 +1,40 @@
+{
+    "slug": "bigpoint",
+    "relevant-countries": [
+        "bg",
+        "cz",
+        "dk",
+        "de",
+        "gr",
+        "gb",
+        "es",
+        "fi",
+        "fr",
+        "hu",
+        "it",
+        "nl",
+        "no",
+        "pl",
+        "pt",
+        "ro",
+        "sk",
+        "se"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Bigpoint GmbH",
+    "address": "Drehbahn 47-48\n20354 Hamburg\nDeutschland",
+    "phone": "+49 40 8814130",
```

Comment:
The privacy policy lists a phone specific to privacy-related requests (+49 821 90786458). Please use that instead ([see](https://github.com/datenanfragen/data/#data-sources)).

Reply:
fixed